cluster/src/Cardano/Cluster.hs

```diff
@@ -52,9 +52,7 @@ import           Pos.Util.CompileInfo (compileInfo, withCompileInfo)
 -- 'Async' handle. For edges nodes, there's an exta connection manager configured
 -- to talk to the underlying node API.
 data RunningNode
-    = RunningCoreNode  NodeName Env (Async ())
-    | RunningRelayNode NodeName Env (Async ())
-    | RunningEdgeNode  NodeName Env Manager (Async ())
```

Comment:
So, since each node now starts its own monitoring API, they all get a `Manager` as a result of setting up their environment. Having three constructors doesn't make much sense anymore so I conflated them into one, but added a `NodeType` argument so that it's still possible to distinguish the kind of node the handles refers to.